pull/context.go

```diff
@@ -200,3 +204,13 @@ type Reviewer struct {
 	Name    string
 	Removed bool
 }
+
+type Signature struct {
+	Email             string
+	IsValid           bool
+	Payload           string
+	Signature         string
+	Signer            string
+	State             string
+	WasSignedByGitHub bool
```

Comment:
Are there other parts of https://docs.github.com/en/graphql/reference/objects#gpgsignature that we want to expose? Maybe `KeyID`?